pkgs/by-name/sp/speedtest/package.nix

```
@@ -0,0 +1,70 @@
+{
+  lib,
+  appstream-glib,
+  blueprint-compiler,
+  desktop-file-utils,
+  fetchFromGitHub,
+  git,
+  glib,
+  gobject-introspection,
+  gtk4,
+  libadwaita,
+  meson,
+  ninja,
+  pkg-config,
+  python3Packages,
+  wrapGAppsHook4,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "speedtest";
+  version = "1.3.0";
+  pyproject = false;
+
+  src = fetchFromGitHub {
+    owner = "Ketok4321";
+    repo = "speedtest";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-A4Pc35SdEo7qIWfcq2s2Bdq+cP1IWi/yrLun3w4FlO8=";
+    leaveDotGit = true;
+  };
+
+  strictDeps = true;
+
+  build-system = [
+    meson
+    ninja
+  ];
+
+  nativeBuildInputs = [
```

```suggestion
  nativeBuildInputs = [
    meson
    ninja
```

(`build-system` is used for python build-time dependencies)